posts/hybrid-cloud-cloud-responsibly.md

```diff
@@ -0,0 +1,94 @@
+---
+authors:
+- Lane Jennison
+- Courtney Parry
+- Rob Zabriskie
+tags:
+- Hybrid
+- Cloud
+- JHipster
+- POC
+date: 2019-08-26T12:00:00.000Z
+title: "Hybric Cloud - Cloud Responsibly"
+image: https://github.com/ippontech/blog-usa/blob/master/images/2019/08/hybrid-cloud.jpg
+---
+
+In this post, we’ll describe the <a href="#concept">hybrid cloud concept</a>, the <a href="#benefits">benefits</a> it provides, and a <a href="#architecture">reference architecture</a> to implementing it.
+
+Moving from on-prem to the cloud via “forklift migration”, where legacy systems are dumped into a cloud-based host, is inefficient and ineffective. The goal should be to strategically extend into the cloud without abandoning the on-prem infrastructure. We do this by incrementally moving services to the cloud, leveraging the advantages of the cloud platform while retaining the value of on-prem where appropriate. This approach allows us to treat cloud vendors as interchangeable based upon cost/value criteria driven by the needs of the business. 
+
+---
+
+<h2 id="concept">Concept</h2>
+Hybrid Cloud is about logically bringing data center environments closer together. Network engineers may view this as the evolution of North-South traffic being treated as East-West. In this context a data center can be a traditional colocation facility, a cloud provider, serverless provider, or SaaS provider. 
+
+As infrastructure technology has improved, the logical boundary where we naturally segment our infrastructure moved. Data center to data center was a natural boundary to designate as WAN traffic for years. Introducing adjacent data centers in close proximity as “availability zones” challenged the idea of a brick-and-mortar data center being the WAN-distance boundary for infrastructure. Regional replication and multi-master technology has further moved the boundary. Now, the only true boundary for infrastructure segmentation is latency.
+
+<h2 id="benefits">What are the benefits?</h2>
+
+- Cost optimization (select the most cost effective environment)
+- Flexibility to host services in separate clouds
+- Take advantage of cloud vendor-specific offerings (security, edge)
+- Option of deployment to cloud and/or on-prem
+- Manage risk of committing to a single vendor
+- Support proximity to customers for location-sensitive applications
+- Assure availability of enterprise-critical services
+
+<h2 id="architecture">Ippon's Approach</h2>
+Our approach allows clients to seamlessly integrate the best features from each cloud into one “best of the breed” infrastructure. Strategically combining and modernizing existing environments while expanding into one or more clouds.  We have created a reference architecture using modern technologies that supports the view of multiple cloud vendors (including on-prem) as a single cloud.  
```

Comment:
I think it is "best of breed"

Reply:
Second sentence does not seem like a full sentence. Maybe start with "We are..."? -> "We are strategically..."